development/dev.env

```diff
@@ -28,3 +28,6 @@ REDIS_PASSWORD=decinablesprewad
 # Needed for Selenium integration tests
 NAUTOBOT_SELENIUM_URL=http://selenium:4444/wd/hub  # WebDriver (Selenium client)
 NAUTOBOT_SELENIUM_HOST=nautobot  # LiveServer (Nautobot server)
+
+# Allow self signed git repositories for config contexts, export templates, ...
+# GIT_SSL_NO_VERIFY="1"
```

Comment:
Might it make sense to document this setting in `docs/configuration/optional-settings.md`?

Reply:
Ah yes, that would make sense.  I will be updating my PR today.